### DISCUSSION TITLE
cluster.check - rpc error

### DISCUSSION THREAD

**Discussion by shuther** [seaweedfs/seaweedfs#5515] [Q&A] [answered]
Could you let me know if the rpc error below is expected or not?
I am using docker compose to launch the service, and I am facing an issue with Kopia -> seaweedfs (with webdav): files are deleted but not sure the origin of the problem yet.

(https://github.com/seaweedfs/seaweedfs)
   command: "master -ip=master -metricsPort=9324 -ip.bind=0.0.0.0 -mdir=/data --volumePreallocate=false -volumeSizeLimitMB=8192 -garbageThreshold=0.01"
V1:    command: 'volume -mserver="master:9333" -ip.bind=0.0.0.0 -port=8080  -metricsPort=9325 -dir=/data -dataCenter=DefaultDataCenter -rack=DefaultRack -disk hdd
V2:   command: 'volume -mserver="master:9333" -ip.bind=0.0.0.0 -port=8080  -metricsPort=9329 -dir=/data -dataCenter=DefaultDataCenter -rack=DefaultRack -disk hdd
filer:     command: 'filer -master="master:9333" -ip.bind=0.0.0.0 -metricsPort=9326 -disk hdd -maxMB=100'
webdav:     command: 'webdav -filer="filer:8888" -maxMB=100'

> cluster.check 
Topology volumeSizeLimit:8192 MB hdd(volume:14/100000 active:14 free:99986 remote:0)
the cluster has 1 filers: [192.168.48.3:8888]
the cluster has 2 volume servers: [192.168.32.3:8080 192.168.32.4:8080]
checking master localhost:9333 to volume server 192.168.32.3:8080 ... ok round trip 1.789ms clock delta -0.064ms
checking master localhost:9333 to volume server 192.168.32.4:8080 ... ok round trip 0.745ms clock delta 0.271ms
checking volume server 192.168.32.3:8080 to master localhost:9333 ... rpc error: code = Unknown desc = ping master localhost:9333: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp [::1]:19333: connect: connection refused"
checking volume server 192.168.32.4:8080 to master localhost:9333 ... rpc error: code = Unknown desc = ping master localhost:9333: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp [::1]:19333: connect: connection refused"
checking filer 192.168.48.3:8888 to master localhost:9333 ... rpc error: code = Unknown desc = ping master localhost:9333: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp [::1]:19333: connect: connection refused"
checking filer 192.168.48.3:8888 to volume server 192.168.32.3:8080 ... ok round trip 0.740ms clock delta 0.271ms
checking filer 192.168.48.3:8888 to volume server 192.168.32.4:8080 ... ok round trip 0.459ms clock delta 0.172ms
checking volume server 192.168.32.3:8080 to 192.168.32.4:8080 ... ok round trip 0.470ms clock delta 0.159ms
checking volume server 192.168.32.4:8080 to 192.168.32.3:8080 ... ok round trip 0.430ms clock delta 0.146ms
checking filer 192.168.48.3:8888 to 192.168.48.3:8888 ... ok round trip 0.119ms clock delta 0.020ms

**Comment by chrislusf** [comment]
configure the master in `weed shell -master=...`

**Comment by shuther** [comment, marked as answer]
thank it seems it was just that. I was expecting that weed sheel ran from the master container would be the same, but it solved it.
